### PR TITLE
upgrade to alpha release of room

### PR DESCRIPTION
**Pull request by amanzanero**
image

### CODE REVIEW

**Comment by amanzanero** [comment]
Current dependencies on/for this PR:
* main
  * **PR #19** <a href="https://app.graphite.dev/github/pr/sebvil/mango/19" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/sebvil/mango/19?utm_source=stack-comment).

File: app/build.gradle.kts [hunk 1]
```diff
@@ -12,12 +12,12 @@ plugins {
 
 android {
     namespace = "com.sebastianvm.mango"
-    compileSdk = 33
+    compileSdk = 34
```
Comment (sebvil) [inline app/build.gradle.kts:15]: Did you not get complains about AGP version?

I had to use `compilePreviewSdk = "UpsideDownCakePrivacySandbox` to get it to work without warnings

File: gradle.properties [hunk 2]
```diff
@@ -20,4 +20,7 @@ kotlin.code.style=official
 # Enables namespacing of each library's R class so that its R class includes only the
 # resources declared in the library itself and none from the library's dependencies,
 # thereby reducing the size of the R class for that library
-android.nonTransitiveRClass=true
\ No newline at end of file
+android.nonTransitiveRClass=true
+# We are opting into the Room Alpha version, which is giving us an "upgrade gradle" warning,
+# but at the time of writing, we are on the latest version.
+android.suppressUnsupportedCompileSdk=34
```
Comment (sebvil) [inline gradle.properties:26]: See my comment above

**Review by sebvil — APPROVED** [review]
Lgtm, left a comment about SDK versions

File: app/build.gradle.kts (hunk 1 above)
Comment (amanzanero) [reply app/build.gradle.kts:15]: yea I did so I added the suppression in the gradle.properties to shut it up 😂

**Pull request merged**